analysis_options.yaml

```diff
@@ -26,7 +26,8 @@ analyzer:
     implicit-dynamic: false
   errors:
     # treat missing required parameters as a warning (not a hint)
-    missing_required_param: warning
+    # TODO: bug #20117
```

Comment:
nit: we usually include the full URL to the bug because they may be in different repositories.